src/components/Dropdown/Items/MultiSelectItem/MultiSelectItem.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import {
+    DropdownMenuCheckboxItem,
+    DropdownMenuCheckboxItemProps,
+} from '@radix-ui/react-dropdown-menu';
+import { bem } from '../../../../utils';
+import { VisualCheckbox } from '../../../Checkbox';
+import { Text } from '../../../Text';
+import styles from './MultiSelectItem.scss';
+
+export interface Props extends Omit<DropdownMenuCheckboxItemProps, 'onChange'> {
+    /** A function to be called if the item is clicked */
+    onChange?: (e: React.SyntheticEvent<HTMLInputElement | HTMLLIElement>) => void;
```

Comment:
we have here `onChange` and `onSelect`, but we use in `MultiSelectItem` stories only `onCheckedChange`, also when we use this component in `Dropdown` story as well, so which one is responsible for what? It's very confusing to me honestly 😅

Reply:
You're right. I replaced the onChange with the onCheckedChange